site/docs/pages/index.mdx

```diff
@@ -4,30 +4,20 @@ description: React components and TypeScript utilities for top-tier onchain apps
 content:
   width: 100%
 layout: landing
-logo: false
+showLogo: false
+showOutline: false
 ---
 
-import { HomePage } from 'vocs/components';
-
 <div className="max-w-[1120px] mx-auto vp-doc relative px-[24px] mb-[96px] mt-[32px] md:px-0 md:mb-[64px]">
-  <div className="pt-[48px] max-sm:pt-0">
-    <div className="max-sm:px-0 flex justify-between z-0 max-md:justify-center">
-      <div className="space-y-8 max-w-[380px] flex flex-col items-start max-md:items-center">
-        <h1 className="text-center text-6xl font-medium no-underline">OnchainKit</h1>
-        <div className="font-regular text-[21px] max-sm:text-[18px] text-[#919193] max-md:text-center">
-          React <span className="text-black dark:text-white">components</span>
-          and TypeScript <span className="text-black dark:text-white">utilities</span>
-          for top-tier onchain apps.
-        </div>
-        <div className="flex justify-center space-x-2">
-          <HomePage.Button href="/getting-started" variant="accent">Get started</HomePage.Button>
-          <HomePage.Button href="https://github.com/coinbase/onchainkit">GitHub</HomePage.Button>
-        </div>
+  <div className="mx-auto max-w-xl">
+    <div className="md:text-center flex flex-col gap-8 pt-48 pb-16">
+      <h1 className="text-center text-8xl font-medium no-underline">OnchainKit</h1>
+      <div className="text-center text-3xl leading-none">
+        <p className="leading-none">React components and TypeScript utilities to help you build top-tier onchain apps.</p>
```

Comment:
you probably want to keep the dark energy here
```
<span className="text-black dark:text-white">
```
for components and utilities in the dark theme.

Reply:
I've addressed this at the theme config directly: https://github.com/coinbase/onchainkit/pull/296/files#diff-43462e5b36aa73561e9f81ebc5ae883c35c8f73e60257d7a0f9aacaa9a34e597R75-R79

felt more appropriate to have consisten theme-level colors for text